.travis.yml

```diff
@@ -5,16 +5,47 @@ php:
   - 5.4
   - 5.5
   - 5.6
-  - 7.0
   - hhvm
-  
+  - nightly
+
 matrix:
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 5.6
+      env: SYMFONY_VERSION=2.3.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.5.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.6.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.x-dev as 2.6"
```

Comment:
I would not make it ``2.6``. 3.0 is not compatible with 2.6 (alias it as ``2.8`` or ``2.9`` if you want such alias, not as ``2.6``)